src/constants/collection/index.ts

```diff
@@ -41,12 +45,14 @@ export const COLLECTION_SORT_ORDER = new Map([
     [CollectionSummaryType.archived, 2],
     [CollectionSummaryType.archive, 3],
     [CollectionSummaryType.trash, 4],
+    [CollectionSummaryType.uncategorized, 4],
 ]);
 
 export const SYSTEM_COLLECTION_TYPES = new Set([
     CollectionSummaryType.all,
     CollectionSummaryType.archive,
     CollectionSummaryType.trash,
+    CollectionSummaryType.uncategorized,
```

Comment:
so, as we discussed, `all`, `archive` and `trash` are just filters, not actuall collections, so they are marked as system collection

but uncategorized is an actual collection and hence it should not be treated as system collection